packages/marble-button/src/Button.js

```diff
@@ -95,6 +95,12 @@ Button.STATE = {
     'default'
   ),
 
+  /**
+   * @default undefined
+   * @type {?(number|undefined)}
+   */
+  tabIndex: Config.number(),
```

Comment:
maybe the default value should be `-1` or `0`. what do you think about that?

Reply:
I thought about it but figured that it makes more sense to leave it undefined, since the default value for links and buttons is implicitly '0'. What do you think?